samples/boards/nrf52/power_mgr_os/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _nrf52-power-mgr-sample:
+
+nRF52 Power management demo
+###########################
+
+Overview
+********
+
+This sample demonstrates a power manager app that uses the Zephyr
+power management infrastructure to enter into Low Power state.
+
+This app will cycle through the following power schemes each time idle thread
+calls _sys_soc_suspend() hook function :
+
+1. Low Power State: Low Power State is SOC specific and being in this state is
+   transparent to devices. SOC and devices do not lose context in this Mode.
+   SOC supports the following two Low Power states :
+
+   A. CONSTANT LATENCY LOW POWER MODE : This Low Power State triggers CONSTLAT
+      task on nrf52 SOC. In this Mode there is Low Exit latency and 16MHz Clock
+      is kept ON.
+   B. LOW POWER STATE : In this Power State LOWPWR task is triggered on nrf52
+      SOC. This is a deeper power state than CONSTANT LATENCY Mode. In this mode
+      the 16MHz Clock is turned off and only 32KHz clock is used.
+
+2. Deep Sleep: This Power State is mapped to SYSTEM OFF state. In this mode
+   all devices on board get suspended. All devices and SOC lose context on
+   wake up.
+
+3. No-op: No operation, letting kernel idle.
+
+Requirements
+************
+
+This application uses nrf52 DK board for the demo.
+
+Building, Flashing and Running
+******************************
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/boards/nrf52/power_mgr
```

Comment:
... and the sample app you're running is in another folder, not the one with this documentation in it.

Reply:
Ok will fix it.